src/components/hooks/use-breakpoints.tsx

```diff
@@ -0,0 +1,28 @@
+import useMediaQuery from './use-media-query'
+import { breakpoints } from 'themes/theme.breakpoints'
+
+export type TUseBreakpoints = {
+    is_mobile: boolean
+    is_tablet: boolean
+    is_lg_tablet: boolean
+    is_laptop: boolean
+    is_desktop: boolean
+}
+
+const useBreakpoints = (): TUseBreakpoints => {
+    const is_mobile = useMediaQuery(breakpoints.xs)
+    const is_tablet = useMediaQuery(breakpoints.sm)
+    const is_lg_tablet = useMediaQuery(breakpoints.md)
+    const is_laptop = useMediaQuery(breakpoints.lg)
+    const is_desktop = useMediaQuery(breakpoints.xl)
+
+    return {
```

Comment:
@michio-rework Maybe it will be too much but we could also return values like `is_mobile_and_tablet` and etc here to avoid having repetitive `&&` or `||` conditions in the code.

Reply:
oh yeah, we surly add those, just wanted to see the feedback, we're starting as simple as possible, but yeah will them thanks.